### PR TITLE
self-hosted to cloud migration addition

### PR DESCRIPTION
**Pull request by justinegeffen**
Added self-hosted to Cloud migration content.

@svelle: please let me know whether I've described the Support process correctly. :)

### CODE REVIEW

**Comment by justinegeffen** [comment]
@svelle, @nab-77; following up on this as I think it's ready to be merged.

File: source/manage/cloud-data-export.rst [hunk 1]
```diff
@@ -136,3 +136,87 @@ Then extract the export file to use it by running the following mmctl command:
...
+
+Before you begin your migration, ensure you’re on Mattermost version 6.5 (or later) as this contains the latest version of the server and the mmctl tool which is used to perform the export. You also need to take note of the following information before you begin:
+
+**User Authentication**
+
+If you’re using a login method other than email, you’ll need to make the required changes to your infrastructure so that you can continue your existing authentication method in the cloud. Examples of these methods are LDAP and GitLab SSO.
+
+**Plugins**
+
+If you’re using plugins that aren’t listed on the Marketplace, they won’t be included in the export and you won’t have access to them going forward.
```
Comment (nab-77) [inline source/manage/cloud-data-export.rst:155]: Can we link out to the KB article that had the plugin list here?

**Review by nab-77 — COMMENTED** [review]
All looks great aside. Can we (as per my comment) link out to the KB article (plugin list)? as The reader has no hints on this page as to what plugins are supported.

File: source/manage/cloud-data-export.rst (hunk 1 above)
Comment (justinegeffen) [reply source/manage/cloud-data-export.rst:155]: We can - do you have the list?

**Comment by justinegeffen** [comment]
@svelle, do you have any thoughts on this?

File: source/manage/cloud-data-export.rst [hunk 2]
```diff
@@ -136,3 +136,87 @@ Then extract the export file to use it by running the following mmctl command:
 Once your migration is complete and you’ve imported your data into your self-hosted instance we recommend that you take a few days to validate your data and ensure everything is working as expected before taking down your Cloud instance.
 
 If you encounter any issues or problems, please contact our Support team via https://customers.mattermost.com/cloud/contact-us, or through the `Mattermost Help Center <https://support.mattermost.com/>`_.
+
+Migrate from self-hosted to Cloud
```
Comment (jasonblais) [inline source/manage/cloud-data-export.rst:140]: Let's add a note that this is only available to paid subscriptions at this time

File: source/manage/cloud-data-export.rst (hunk 1 above)
Comment (svelle) [reply source/manage/cloud-data-export.rst:155]: here's the link https://support.mattermost.com/hc/en-us/articles/5346624843924-Mattermost-Plugin-Marketplace-List-as-of-April-5th-2022

**Comment by svelle** [comment]
Hey @justinegeffen nothing to add aside from what @nab-77 and @jasonblais already pointed out.

**Comment by justinegeffen** [comment]
@svelle, @nab-77: List link has been added. If one of you could approve this PR then we can merge it. :)

**Review by nab-77 — APPROVED** [review]
Perfect addition to the migration page. Thank you @justinegeffen.

**Pull request merged**